scripts/globals/pets/wyvern.lua

```diff
@@ -59,28 +60,29 @@ function onMobSpawn(mob)
         mob:addJobTraits(master:getSubJob(), master:getSubLvl())
     end
     local wyvernType = wyvernTypes[master:getSubJob()]
-    local healingbreath = 624
-    if mob:getMainLvl() >= 80 then healingbreath = 623
-    elseif mob:getMainLvl() >= 40 then healingbreath = 626
-    elseif mob:getMainLvl() >= 20 then healingbreath = 625 end
+    local healingbreath = tpz.jobAbility.HEALING_BREATH
+    if mob:getMainLvl() >= 80 then healingbreath = tpz.jobAbility.HEALING_BREATH_IV
+    elseif mob:getMainLvl() >= 40 then healingbreath = tpz.jobAbility.HEALING_BREATH_III
+    elseif mob:getMainLvl() >= 20 then healingbreath = tpz.jobAbility.HEALING_BREATH_II
+	end
```

Comment:
Haven't tested this code, but have some styling notes.  (See style guide for contributions [here](https://github.com/project-topaz/topaz/blob/release/CONTRIBUTING.md).)

Set editor to use 4 spaces for indentation, rather than tabs.  Tabs muck up the indentation for reviewers in github.

Reply:
Gotcha, okay, I'll use 4 spaces going forward :+1: